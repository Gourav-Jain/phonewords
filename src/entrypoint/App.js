import React from "react";

function App() {
  return (
    <div className="App">
      <h1>Phonewords</h1>
    </div>
  );
}

export default App;
