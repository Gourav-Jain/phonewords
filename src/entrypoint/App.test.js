import React from "react";
import { render } from "@testing-library/react";
import App from "./App";

jest.mock("react-redux", () => ({
  useDispatch: jest.fn,
  useSelector: () => ({
    phoneword: {
      showLoader: false,
      error: false,
      listOfWords: [],
    },
  }),
}));

test("@App", () => {
  const { container } = render(<App />);
  expect(container).toMatchSnapshot();
});
