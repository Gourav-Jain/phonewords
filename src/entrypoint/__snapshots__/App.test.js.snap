// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@App 1`] = `
<div>
  <div
    class="container"
  >
    <div
      class="top"
    >
      <h2>
        Phoneword
      </h2>
      <input
        class="text-input"
        name="phonenumber"
        readonly=""
        type="text"
        value=""
      />
      <div
        class="inner-container"
      >
        <div
          class="input"
        >
          <span>
            1
          </span>
          <p
            class="small-input"
            id="sub-char"
          />
        </div>
        <div
          class="input"
        >
          <span>
            2
          </span>
          <p
            class="small-input"
            id="sub-char"
          >
            abc
          </p>
        </div>
        <div
          class="input"
        >
          <span>
            3
          </span>
          <p
            class="small-input"
            id="sub-char"
          >
            def
          </p>
        </div>
      </div>
      <div
        class="inner-container"
      >
        <div
          class="input"
        >
          <span>
            4
          </span>
          <p
            class="small-input"
            id="sub-char"
          >
            ghi
          </p>
        </div>
        <div
          class="input"
        >
          <span>
            5
          </span>
          <p
            class="small-input"
            id="sub-char"
          >
            jkl
          </p>
        </div>
        <div
          class="input"
        >
          <span>
            6
          </span>
          <p
            class="small-input"
            id="sub-char"
          >
            mno
          </p>
        </div>
      </div>
      <div
        class="inner-container"
      >
        <div
          class="input"
        >
          <span>
            7
          </span>
          <p
            class="small-input"
            id="sub-char"
          >
            pqrs
          </p>
        </div>
        <div
          class="input"
        >
          <span>
            8
          </span>
          <p
            class="small-input"
            id="sub-char"
          >
            tuv
          </p>
        </div>
        <div
          class="input"
        >
          <span>
            9
          </span>
          <p
            class="small-input"
            id="sub-char"
          >
            wxyz
          </p>
        </div>
      </div>
      <div
        class="inner-container"
      >
        <div
          class="input"
        >
          <span>
            *
          </span>
          <p
            class="small-input"
            id="sub-char"
          />
        </div>
        <div
          class="input"
        >
          <span>
            0
          </span>
          <p
            class="small-input"
            id="sub-char"
          />
        </div>
        <div
          class="input"
        >
          <span>
            #
          </span>
          <p
            class="small-input"
            id="sub-char"
          >
            Clear
          </p>
        </div>
      </div>
      <div
        class="inner-container"
      >
        <button
          class="btn"
          disabled=""
        >
          Click & Get Word List
        </button>
      </div>
    </div>
  </div>
</div>
`;
